push-sender/src/main/java/org/jboss/aerogear/unifiedpush/message/sender/fcm/ConfigurableFCMSender.java

```diff
@@ -36,11 +36,15 @@ public ConfigurableFCMSender(String key) {
 
 	@Override
 	protected HttpURLConnection getConnection(String url) throws IOException {
-		
+
```

Comment:
please refrain from unneeded white space changes

Reply:
This is the replacement of a unneeded tabbed line with a clean newline.

Reply:
OK